packages/frontend/app/dashboard/components/admin-dashboard.tsx

```diff
@@ -0,0 +1,84 @@
+import { useRouter } from "next/navigation";
+
+/*
+Just for now before I can figure out how to programatically generate these
+dashboards based on the user type
+*/
+/* eslint-disable react/jsx-key */
```

Comment:
Best to remove this, we need jsx keys for react to render fast because react can cache frequent renders using a provided key in a JSX element.